docs/en/operations/table_engines/odbc.md

```diff
@@ -0,0 +1,29 @@
+# ODBC {#table_engine-odbc}
+
+Allows ClickHouse to connect to external databases via [ODBC](https://en.wikipedia.org/wiki/Open_Database_Connectivity).
+
+To implement ODBC connection, ClickHouse uses the separate program `clickhouse-odbc-bridge`. ClickHouse starts this program automatically when it is required. The ODBC bridge program is installed by the same package with the ClickHouse server.
+
+This engine supports the [Nullable](../../data_types/nullable.md) data type.
```

Comment:
Arrays are probably also worth mentioning

Reply:
Arrays are not the special case. Aren't them?

Reply:
they are even more special (less common)